src/aggregator/src/main/java/au/org/emii/aggregator/template/TemplateDataset.java

```diff
@@ -158,9 +159,15 @@ public TemplateDataset(NetcdfDatasetIF dataset, AggregationOverrides aggregation
             result.put("TIME_END", timeRange.getEnd().toString());
         }
 
-        if (verticalRange != null) {
-            result.put("Z_MIN", Integer.toString(verticalRange.first()));
-            result.put("Z_MAX", Integer.toString(verticalRange.last()));
+        if (dataset.hasVerticalAxis()) {
+            if (verticalRange != null) {
+                result.put("Z_MIN", Integer.toString(verticalRange.first()));
+                result.put("Z_MAX", Integer.toString(verticalRange.last()));
+            } else {
+                Bounds verticalBounds = dataset.getVerticalAxis().getBounds();
+                result.put("Z_MIN", Integer.toString((int)verticalBounds.getMin()));
```

Comment:
Actually just noticed the (int)  -  looks to me like getBounds returns the min/max values as a double whereas verticalRange.first and verticalRangeLast returns the min/max index - probably should be returning the same type of thing - and I would have thought the value would be more relevant than the index for display in the global attributes.

Reply:
Tested this in prod and the values are coming through OK, but we shouldn't be coercing the value to an integer - this is CARS specific.

Reply:
Ah sorry @jonescc, missed your first comment. Doesn't verticalRange.first() and last() return the min/max value rather than the index? When I tested this in my own stack, that's what I got back in the attributes. But that doesn't seem to make sense... given that Range is a range of indexes. I'm a bit confused!

Reply:
First comment was incorrect are getting values.  However, there is an issue with converting to int. 
 For CARS values are int, other datasets may not be.   Conversion should happen when applying global attribute overrides if necessary.  This is an existing bug so I'm raising an issue for it.  No need to address here.